Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -294,3 +294,126 @@ angular.module("umbraco").controller("customBlockController", function ($scope,
     });    
 });
 ```
+
+## Creating Blocklist programmatically
+
+In this example we are creating some Blocklist objects under the `People` property in `Home` Document Type. The `People` property implements a Blocklist datatype where a `Person` Document Type can be created. The `Person` Document Type has two properties - `user_name` and `user_email`.
+
+The approach to saving Blocklist content programmatically is similar to Nested Content - though the JSON schema is a bit different.
```

Comment:
Consider if this is relevant for someone who never has and never will work with Nested Content.